graphqlapi/src/main/java/io/stargate/graphql/web/HttpAwareContext.java

```diff
@@ -41,9 +43,10 @@
   // For more information.
   private final BatchContext batchContext = new BatchContext();
 
-  public HttpAwareContext(HttpServletRequest request) {
+  public HttpAwareContext(HttpServletRequest request, Persistence persistence) {
```

Comment:
This is more convenient than modifying all the fetcher constructors.
I think we should do the same for existing constructor arguments, see #945.